src/xref2/test/lib/common.cppo.ml

```diff
@@ -45,10 +45,24 @@ let root =
         ~module_name:"Root"
         ~digest:"nodigest"
 
-let root_identifier = `Identifier (`Root (root, Odoc_model.Names.UnitName.of_string "Root"))
+let root_with_name = `Root (root, Odoc_model.Names.UnitName.of_string "Root")
+
+let root_identifier = `Identifier root_with_name
+
+let root_module name = `Module (root_with_name, Odoc_model.Names.ModuleName.of_string name)
 
 let root_pp fmt (_ : Odoc_model.Root.t) = Format.fprintf fmt "Common.root"
 
+let _quoted fmt_a fmt a = Format.fprintf fmt "\"%a\"" fmt_a a
+let _quoted_str a_to_str fmt a = Format.fprintf fmt "%S" (a_to_str a)
+
+let unit_name_pp = _quoted_str Odoc_model.Names.UnitName.to_string
```

Comment:
There are formatters for these - `Odoc_model.Names.UnitName.fmt` - we could just use those (which would drop the quote marks though)

Reply:
Some of these are of module type `SimpleName` which doesn't have that function.

Reply:
We should probably add `fmt` to `SimpleName`

Reply:
I added `fmt` and used it directly, dropping the quotes. The diffs are bigger but that's not important I think